### DISCUSSION TITLE
How does the new "errors" prop work with useForm

### DISCUSSION THREAD

**Discussion by AlemTuzlak** [react-hook-form/react-hook-form#11295] [Q&A] [answered]
Hello! 👋 
I'm the maintainer of remix-hook-form and I'd love to understand how does errors prop work. 

What I'm doing atm in remix-hook-form is creating the same errors shape on the backend as in the frontend using resolver(data). I send it to the frontend and then I merge the errors together, obviously with this new `errors` prop I'd assume I can just give it the backend errors and I'm good. I also got reports from people that the reset function doesn't work with my approach so I'm wondering if I switch to this would it work identically to react-hook-form?

Thank you in advance!

**Comment by Moshyfawn** [comment, marked as answer]
Hi Alem 👋

If you haven't checked #11188 already, it contains a decent explanation of the problem the new `errors` API aims to solve.

Let me know if you have any further questions.

**Reply by AlemTuzlak** [reply]
@Moshyfawn this is exactly what I needed to know and works exactly as I hoped, this will simplify my remix-hook-form code a lot, thank you
